changelog

```diff
@@ -1,3 +1,20 @@
+Version 0.1.2
+----------------------------
+1. GDPR compliance to Telemetry and Logger
+2. Fix the bug on CustomTabService checking when calling getCustomTabsServiceIsBound()
+3. Update Telemetry to track telemetry Event data in a single Map instance
+4. Specifies thrown Exception classes in AbstractMetadataRequestor
+5. Create "no-args" constructor for GSON POJOs to prevent using "sun.misc.Unsafe" 
+6. Specifies the default locale to UTF_8 to guarantee consistent behavior across all Android devices.
+7. Use versions.gradle to aggregate versions across subproject
+8. Update Gradle to 4.1
+9. Update Gson to 2.8.0
+
+Version 0.1.1
+----------------------------
+* This release contains the preview for Microsoft Authentication library (MSAL) for Android.
```

Comment:
why is 0.1.1 showing up here? was this missed from the change log?

Reply:
Yes, it is missed in the previous release.